README.md

```diff
@@ -165,12 +165,16 @@ Some common props that is available on all of the components
 * [SceneContainer](#sceneContainer)
 
 #### Button
-| Prop       | Description                                                          | Type       |
+| Prop        | Description                                                          | Type       |
 |------------|----------------------------------------------------------------------|------------|
-| `variant`  | String that should be one of the defined colors in the color.js file | `string`   |
-| `onPress`  | Assigning button trigger                                             | `function` |
-| `loading`  | Shows a loading indicator in the middle of the button                | `boolean`  |
-| `disabled` | Disables the button                                                  | `boolean`  |
+| `variant`   | String that should be one of the defined colors in the color.js file | `string`   |
+| `onPress`   | Assigning button trigger                                             | `function` |
+| `loading`   | Shows a loading indicator in the middle of the button                | `boolean`  |
+| `disabled`  | Disables the button                                                  | `boolean`  |
+| `textStyle` | Basic inline styling                                                 | `boolean`  |
```

Comment:
Should probably be `Text styles` and a link to https://facebook.github.io/react-native/docs/text-style-props